lib/sync/dataHandlers.js

```diff
@@ -259,6 +261,16 @@ module.exports = function(options) {
       handler(datasetId, dataHash, metaData, callback);
     },
 
+    /**
+     * @param datasetId
+     * @param metaData
+     * @param callback
+     */
+    countCollisions: function(datasetId, metaData, callback) {
```

Comment:
@wei-lee I'm probably missing something but I can't find a place where this should/could be done? We find if there is a collision in the pending worker, but we don't seem to have a way to access the dataset client from there in order to update it?